### ISSUE TITLE
App getting crash on iOS-14 once user tap on Push Notification.

### ISSUE THREAD

**Issue by rizwan9787** [infobip/mobile-messaging-sdk-ios#23]
We are using bellow configuration and the app is crashing every time while user clicks on the push notification so kindly look into it.

It's crashing at
```
public class func didReceiveRemoteNotification(_ userInfo: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void)  {
        
        guard let mm = MobileMessaging.sharedInstance else {
            completionHandler(UIBackgroundFetchResult.failed)
            return
        }
        
        mm.didReceiveRemoteNotification(userInfo, completion: { result in
            completionHandler(result.backgroundFetchResult)
        })
    }
```
File:
`MobileMessaging.swift`

line where app is crash:
`completionHandler(result.backgroundFetchResult)`

With reason:
`Thread 6: EXC_BREAKPOINT (code=1, subcode=0x108b6efd8)`

* OS Version: iOS-14
* MobileMessaging SDK version 8.3.1
* Xcode version: 12
* Your application language: Swift 4.2
* Cocoapods version: 1.9.3

**Comment by kas-kad** [comment]
Hi, thanks for the report, we will check and fix it shortly. Will keep you updated.

**Comment by kas-kad** [comment]
@rizwan9787 I could not reproduce the crash for the conditions you mentioned. Could you provide more details on the exception you have?

**Comment by rizwan9787** [comment]
Okay let me check with your demo app first, then I'll let you know the result.

**Comment by habibaamri** [comment]
Please did you find a solution , I have the same pro my app crash after publication on TestFlight.

**Comment by kas-kad** [comment]
@habibaamri hi. Which version of SDK do you use?

**Comment by kas-kad** [comment]
The crash was fixed in 8.3.3 and 6.2.7. Closing this one due to inactivity.

**Comment by mudassarahmad09** [comment]
I am having the exact same issue.  I am using the MobileMessaging SDK version 10.17.0. I am also attaching the image.
<img width="1715" alt="Screenshot 2023-06-07 at 12 34 22 AM" src="https://github.com/infobip/mobile-messaging-sdk-ios/assets/34540384/39d08c5b-bc13-4301-9c42-0c3e851e70c4">
